docs/operate-and-deploy/monitoring.md

```diff
@@ -0,0 +1,81 @@
+# Monitoring
+
+## Context
+
+ksqlDB publishes metrics via JMX ([Java Management
+Extensions](https://www.oracle.com/java/technologies/javase/javamanagement.html))
+which help you monitor what is happening inside of ksqlDB's servers. A
+comprehensive [set of metrics](../reference/metrics.md) are published,
+including number of rows processed, throughput,
```

Comment:
throughput distribution, and error rate, see [Metrics](../reference/metrics.md).